addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -122,6 +122,13 @@ const DynamicSnippet = publicWidget.Widget.extend({
                     'limit': parseInt(nodeData.numberOfRecords),
                     'search_domain': this._getSearchDomain(),
                     'with_sample': this.editableMode,
+                    'context': {
```

Comment:
Given the TB on runbot, I guess this has/can now be set as we expected (not in params)?
Seems like right here, it's shadowing the lower level context which is added to every rpc (website/lang).

Reply:
That is a bug in httpocalypse, solved in the commit I just pushed on the branch :)